database/tests/unit/test_events.py

```diff
@@ -1,8 +1,6 @@
 import os
 
-import pytest
-
-import database.events
+import database.events  # noqa: F401
```

Comment:
Soo, I searched this up and this F401 is an unused imported module, https://www.flake8rules.com/rules/F401.html#:~:text=A%20module%20has%20been%20imported,the%20import%20should%20be%20removed. Do we want to get rid of this line as part of running ruff? Or do we just want to signal it?

(I think we should remove it, maybe it was missed?)

Reply:
Interesting.. that's werid haha, I guess ruff can tell? Maybe there's methods from database.events that are silently used, but ya that's odd